mtp_cashbook/templates/cashbook/includes/table_actions.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<div class="batch-actions">
+  <p class="print">
+    <a href="#print-dialog" class="js-Dialog">{% trans "Print these transactions" %}</a>
+  </p>
+  <p class="select-all">
+    <input type="checkbox" id="select-all-{{ location }}" class="Checkbox--right js-SelectAll" data-name="{{ field_name }}">
+    <label for="select-all-{{ location }}">{% trans "Select all" %}</label>
+  </p>
+</div>
```

Comment:
As discussed yesterday, I'm not about this right now. 

Without JS they are currently hidden from the view and moving them to JS would either mean having the markup in JS files or adding JS templating, which might increase the complexity around internationalization. 

I'm happy either way but at the moment leaning towards what we have and not involving JS templates as it keeps all the i18n handled by django. 

What's your preference?